jre_emul/misc_tests/com/google/j2objc/security/SecurityTests.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.j2objc.security;
+
+import com.google.j2objc.TestUtil;
+import junit.framework.Test;
+import junit.framework.TestSuite;
+
+/**
+ * Returns a suite of all java.security.spec tests.
+ */
+public class SecurityTests {
+
+    public static Test suite() {
+        TestSuite suite = new TestSuite();
+        suite.addTest(TestUtil.getPackageTests("tests.security.spec"));
```

Comment:
Since you named the class "SecurityTests", add the other security test packages? It shouldn't take that much longer to run, and when doing security work might find a bug not directly related to the code being updated.

Reply:
Done